k8s/namespaces/probate/probate-cron-make-dormant-cases/demo.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-cron-make-dormant-cases
+spec:
+  releaseName: probate-cron-make-dormant-cases
+  values:
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: demo
+    job:
+      memoryRequests: "1024Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "2048Mi"
+      cpuLimits: "2000m"
+      image: hmctspublic.azurecr.io/probate/back-office:pr-2018-f73d655-20220817104223 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
+      schedule: 0/10 * * * *
```

Comment:
do you want this to run every 10 minutes in demo?

Reply:
For testing purpose I have added 10 mins as suggested by Sunil